app/services/import_polygons.rb

```diff
@@ -41,4 +43,51 @@ def location_categories_include?(region_name)
       location_type == :london_boroughs && DOWNCASE_BOROUGHS.include?(region_name) ||
       location_type == :cities && DOWNCASE_CITIES.include?(region_name)
   end
+
+  def get_buffers(points)
+    # ArcGIS API documentation: https://ons-inspire.esriuk.com/arcgis/sdk/rest/index.html#//02ss0000003z000000
+    buffer_distances_in_miles = [5, 10, 15, 20, 25]
+    buffered_boundaries = {}
+    buffer_distances_in_miles.each do |distance|
+      # convert 1D array, to 2D array for arcgis
+      polygon_coords = points.each_slice(2).to_a
+      response = HTTParty.get(buffer_api_endpoint(polygon_coords, convert_miles_to_metres(distance)))
+      buffer_coords = response.dig("geometries", 0, "rings").flatten
+      # algolia requires a 2D containing 1 element with all the coordinates contained within it
+      # so buffer_coords array has been wrapped in another array
```

Comment:
Does this wrapping happen here or elsewhere? It's confusing to talk about some wrapping that happens elsewhere.

Reply:
The wrapping happens in the location builder. I've now remove the comment about the wrapping and have kept a comment mentioning that the buffer coordinates are stored as a 1D array.